src/transforms/rotate.jl

```diff
@@ -35,7 +35,7 @@ function Rotate(u::Vec, v::Vec)
   u⃗ = normalize(u)
   v⃗ = normalize(v)
   re = √((1 + u⃗ ⋅ v⃗)/2)
-  im = (u⃗ × v⃗) / 2re
+  im = (v⃗ × u⃗) / 2re
```

Comment:
Is this a bug in our Rotate transform? If yes, we need to fix it in a separate PR.

Reply:
No, that's still due to the fact that `ReferenceFramesRotation` rotates the frame.

Reply:
I'm tired I didn't sleep... In fact I think there's a problem: in our `Rotate` stuff we inverse the rotation. But here I don't use our `Rotate` stuff... So I think this rotation will be inverted if we apply it to a mesh. But me I don't apply it to a mesh, you see?

Reply:
Yes, I think we need a new utility function in utils.jl that takes two vectors and returns the appropriate DCM for the rotation of the object. We can then use this utility function in the constructor of Rotate.

Reply:
I've added a new utility function called `uvrotation` with the code from Rotate. You can now rebase this PR on top of the latest master branch to make use of it.